src/main/kotlin/coverosR3z/timerecording/api/ViewTimeAPI.kt

```diff
@@ -238,7 +261,9 @@ class ViewTimeAPI(private val sd: ServerData) {
         val timeEntry = TimeEntry(entryId, employee, project, time, date, details)
         tru.changeEntry(timeEntry)
 
-        return redirectTo(path)
+        val currentPeriod = data.mapping[Elements.TIME_PERIOD.getElemName()]
+        val viewEntriesPage = path + if (currentPeriod.isNullOrBlank()) "" else "" + "?" + Elements.TIME_PERIOD.getElemName() + "=" + currentPeriod
```

Comment:
This is an example of code where we put the new query string for showing a particular time period